README.md

```diff
@@ -65,6 +65,22 @@ git-credential-gopass configure --local --store=ci-team
 
 This puts the value in front of the Gopass search path.
 
+#### Using with SMTP
+
+If you want to use this with [`git-send-email`](https://git-scm.com/docs/git-send-email) you'll need to:
+
+1. [Set Git Credential Helper](#set-git-credential-helper)
+2. Put your password in `.../gopass/stores/root/git/example.com_465/username.gpg`
```

Comment:
I would suggest to use the secret name here, that is `git/example.com_465/usename`.

Reply:
Yeah that might be a better wording, not literal password but secret that is encrypted.